src/App/components/MediaObject/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const MediaObject = ({ size, imgUrl, icon, heading, text, mediaRight }) => {
+    const mediaHeading = () => {
+        switch (size) {
+            case "sm":
+                return <h4>{heading}</h4>;
+            case "lg":
+                return <h2>{heading}</h2>;
+            default:
+                return <h3>{heading}</h3>;
+        }
+    };
+
+    return (
+        <div className={`media${mediaRight ? " media-right" : ""}${size ? ` media-${size}` : ""}`}>
+            {(imgUrl || icon) ?
+                <div className="media-img">{"\n"}
```

Comment:
Not sure I follow why the newline is here, is it because of how it is rendered in the documentation page?